examples/gno.land/r/foo20/foo.gno

```diff
@@ -22,39 +22,39 @@ func init() {
 
 // getters.
 
-func TotalSupply() uint64                         { return foo.TotalSupply() }
-func BalanceOf(address std.Address) uint64        { return foo.BalanceOf(address) }
-func Allowance(owner, spender std.Address) uint64 { return foo.Allowance(owner, spender) }
+func TotalSupply() uint64                    { return foo.TotalSupply() }
+func BalanceOf(address string) uint64        { return foo.BalanceOf(address) }
+func Allowance(owner, spender string) uint64 { return foo.Allowance(owner, spender) }
 
 // setters.
 
-func Transfer(to std.Address, amount uint64) {
-	caller := std.GetCallerAt(2)
+func Transfer(to string, amount uint64) {
```

Comment:
why not AddressOrName?

Reply:
Now it’s done, I should have put the PR in draft :)